src/gp-model/main.py

```diff
@@ -55,92 +22,54 @@ def get_airqloud_polygon(tenant, airqloud):
     params = {'tenant': tenant,
               'name': airqloud
               }
-    coords = requests.get(configuration.VIEW_AIRQLOUD_URI, params=params).json()[
-        'airqlouds'][0]['location']['coordinates']
+    if environment == 'staging':
+        headers = {'Authorization': configuration.API_TOKEN}
```

Comment:
We also have a new authentication method that use Access token passed as a query parameter which I find to be a cleaner approach.

Reply:
@123MwanjeMike done.

Reply:
Great, thanks @uman95